### PR TITLE
UBI Packer / Unpacker

### PR DESCRIPTION
**Pull request by kibo-redballoon**
Please describe the changes in your request.
This PR contributes a UBI packer / unpacker and associated tests for OFRAK. This is still WIP.

Anyone you think should look at this, specifically?
@whyitfor

### CODE REVIEW

**Review by EdwardLarson — CHANGES_REQUESTED** [review]
Update changelog por favor

**Review by EdwardLarson — APPROVED** [review]

File: ofrak_core/ofrak/core/ubi.py [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+@dataclass
+class UbiVolume(ResourceView):
+    """
+    An UbiVolume is a volume entry in UBI. It typically contains an image of arbitrary data, typically a filesystem or a
+    log. They can be empty while reserving PEBs for future volume modification or for padding.
+
+    Volume information reflected in the 'config.ini' UBI volume entries expected by `ubinize` are stored here. Also see:
+    http://www.linux-mtd.infradead.org/faq/ubi.html#L_ubi_mkimg and
+
+    :cvar id: The assigned volume ID within the UBI image
```
Comment (whyitfor) [inline ofrak_core/ofrak/core/ubi.py:46]: Have you tested that `cvar` renders correctly in the built docs? I know that `var` and `ivar` both work

I think technically these are not class variables -- it looks like class variables are specifically skipped by dataclass: https://docs.python.org/3/library/dataclasses.html#class-variables. 

Let's just change all of these to `:var ...`?

**Review by whyitfor — CHANGES_REQUESTED** [review]
Looks good!

See comment about dataclass docstrings.

**Review by whyitfor — APPROVED** [review]
LGTM!

**Pull request merged**